src/renderer/components/+catalog/catalog.tsx

```diff
@@ -180,6 +180,7 @@ export class Catalog extends React.Component<Props> {
         material={item.entity.spec.icon?.material}
         background={item.entity.spec.icon?.background}
         onClick={() => this.onDetails(item)}
+        interactive={false}
```

Comment:
This seems strange, given that we also have an `onClick` defined.

Reply:
Removed this prop.